ghlicense/cmd.py

```diff
@@ -204,48 +208,32 @@ def pick_license_from_last_used(last_used_licenses):
-                     '\tAny modifications must also be made available under\n'
-                     '\tthe GPL along with build & install instructions.'
-                     '\n\n  GPLv3\n'
-                     '\tSame of GPLv2 but easily integrable with other licenses.'
-                     '\n\n  LGPLv3\n'
-                     '\tThis license is mainly applied to libraries.\n'
-                     '\tDerivatives works that use LGPL library can use other licenses.'
-                     '\n\n  AGPLv3\n'
-                     '\tThe AGPL license differs from the other GNU licenses in that it was\n'
-                     '\tbuilt for network software, the AGPL is the GPL of the web.'
-                     '\n\n  FDLv1.3\n'
-                     '\tThis license is for a manual, textbook, or other\n'
-                     '\tfunctional and useful document "free" in the sense of freedom.'
-                     '\n\n  Apachev2\n'
-                     '\tYou can do what you like with the software, as long as you include the\n'
-                     '\trequired notices.'
-                     '\n\n  CC-BY\n'
-                     '\tThis is the ‘standard’ creative commons.\n'
-                     '\tIt should not be used for the software.'
-                     '\n\n  BSDv2\n'
-                     '\tThe BSD 2-clause license allows you almost unlimited freedom.'
-                     '\n\n  BSDv3\n'
-                     '\tThe BSD 3-clause license allows you almost unlimited freedom.'
-                     '\n\n  BSDv4\n'
-                     '\tThe BSD 4-clause license is a permissive license with a special \n'
-                     '\tobligation to credit the copyright holders of the software.'
-                     '\n\n  MPLv2\n'
-                     '\tMPL is a copyleft license. You must make the source code for any\n'
-                     '\tof your changes available under MPL, but you can combine the\n'
-                     '\tMPL software with proprietary code.'
-                     '\n\n  UNLICENSE\n'
-                     '\tReleases code into the public domain.'
-                     '\n\n  MIT\n'
-                     '\tA short, permissive software license.'
-                     '\n\n  EUPL\n'
-                     '\tThe “European Union Public Licence” (EUPL) The EUPL is the first\n'
-                     '\tEuropean Free/Open Source Software (F/OSS) licence. It has been\n'
-                     '\tcreated on the initiative of the European Commission.\n\n')
+    licenses_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'licenses.txt')
+    try:
+        with open(licenses_path, 'r') as file:
+            licenses = file.read()
+            sys.stderr.write(f'{licenses}\n')
+    except FileNotFoundError:
+        sys.stderr.write('licenses.txt file not found.\n')
+
+
+if __name__ == '__main__':
+    print_license_list()
+
+print_license_list()
```

Comment:
Why this lines?
This function is executed on line https://github.com/Mte90/GH-License/blob/b9a3539000e89a05d50afa218ae737607ba07fd4/ghlicense/cmd.py#L340 so aren't needed